docs/source/using/troubleshooting.adoc

```diff
@@ -11,6 +11,26 @@ toc::[]
 
 This section contains solutions to common problems that you might encounter while using Minishift.
 
+[[minshift-update-failed-due-to-permission-denied]]
+== Permission cause update to fail
+
+Depending on your operating system if you have minishift binary installed in location where current user does not have write permission then `minishift update` will fail.
+
```

Comment:
It is not a different issue. In fact the error we are seeing is because the user does not have permissions to write to the directory. Have a closer look at the error message. The binary itself + the directory need to be writable to the user executing the update. 

> Otherwise how the minishift binary is present in the location.

Because you installed it as root.